### ISSUE TITLE
Review: 1.4.6 Contrast (Enhanced) (AAA)

### ISSUE THREAD

**Issue by qbalsdon** [GetEvinced/mcag#28]
Task: Edit [MCAG](https://github.com/GetEvinced/mcag/blob/main/src/app/data/mcag.json) section 1.4.6 considering the mobile research questions (to be completed)

---
The visual presentation of text and images of text has a contrast ratio of at least 7:1, except for the following:

    Large Text: Large-scale text and images of large-scale text have a contrast ratio of at least 4.5:1;

    Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.

    Logotypes: Text that is part of a logo or brand name has no contrast requirement.
---
Sufficient Techniques for Success Criterion 1.4.6

Note: Other techniques may also be sufficient if they meet the success criterion. See [Understanding Techniques.](https://www.w3.org/TR/2014/NOTE-UNDERSTANDING-WCAG20-20140916/understanding-techniques.html#understanding-techniques)
Situation A: text is less than 18 point if not bold and less than 14 point if bold

    G17: Ensuring that a contrast ratio of at least 7:1 exists between text (and images of text) and background behind the text
    G148: Not specifying background color, not specifying text color, and not using technology features that change those defaults
    [G174: Providing a control with a sufficient contrast ratio that allows users to switch to a presentation that uses sufficient contrast ](https://www.w3.org/WAI/WCAG22/Techniques/general/G174.html)

Situation B: text is as least 18 point if not bold and at least 14 point if bold

    G18: Ensuring that a contrast ratio of at least 4.5:1 exists between text (and images of text) and background behind the text
    G148: Not specifying background color, not specifying text color, and not using technology features that change those defaults
    [G174: Providing a control with a sufficient contrast ratio that allows users to switch to a presentation that uses sufficient contrast ](https://www.w3.org/WAI/WCAG22/Techniques/general/G174.html)

Advisory Techniques for Success Criterion 1.4.6

    [G156: Using a technology that has commonly-available user agents that can change the foreground and background of blocks of text ](https://www.w3.org/WAI/WCAG22/Techniques/general/G156.html)

Failures for Success Criterion 1.4.6

    F24: Failure of Success Criterion 1.4.3, 1.4.6 and 1.4.8 due to specifying foreground colors without specifying background colors or vice versa
    F83: Failure of Success Criterion 1.4.3 and 1.4.6 due to using background images that do not provide sufficient contrast with foreground text (or images of text)
